web_services/foxx/brightid/db.js

```diff
@@ -899,54 +899,83 @@ function getSponsorship(contextId) {
 
 function sponsor(op) {
```

Comment:
this function should be changed in a way that support both old and new sponsor operations